chrome/content/zotero/integration/integrationDocPrefs.xul

```diff
@@ -77,4 +84,4 @@
 		<checkbox id="storeReferences" label="&zotero.integration.prefs.storeReferences.label;"/>
 		<description class="radioDescription">&zotero.integration.prefs.storeReferences.caption;</description>
 	</vbox>
-</dialog>
\ No newline at end of file
+</dialog>
```

Comment:
What would be the git flow for that? I probably would like to keep your and my commits separate once we squash this PR, so how would that work?